README.md

```diff
@@ -24,7 +24,7 @@ More Resources
 
 -   [Official turboWAVE Repo](https://github.com/USNavalResearchLaboratory/turboWAVE)
 -   [TurboWAVE Documentation](https://turbowave.readthedocs.io)
--   The current html turboPy documentation can be found [here](https://turbopy.readthedocs.io/en/latest)
+-   The published paper for turboPy documentation can be found [here](https://doi.org/10.1016/j.cpc.2020.107607)
```

Comment:
suggestion:
Instead of "The published paper for turboPy documentation can be found"
try "The published turboPy paper can be found"

Reply:
Sure!